src/util.ts

```diff
@@ -104,3 +105,11 @@ export async function getRandomUserAgent(): Promise<string> {
   logger.debug('user agent', {userAgent});
   return userAgent;
 }
+
+export function deleteFile(path: string) {
```

Comment:
**suggestion(non-blocking):** As previously mentioned, it could be easily to put into a temp directory. But this is fine for now as I don't want to block this.